common/src/macros/oops.rs

```diff
@@ -7,8 +7,8 @@
 /// ```
 #[macro_export]
 macro_rules! oops {
-    ($($arg:tt)*) => {
+    ($($arg:tt)*) => {{
```

Comment:
this change should be fine in a separate commit

Reply:
i close pull request and send again please review